app/models/bgs_dependents/adult_child_attending_school.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module BGSDependents
+  class AdultChildAttendingSchool < Base
```

Comment:
Please document class attributes and methods with [YARD](https://yardoc.org/)

Reply:
Added comments for YARD documentation.